apps/cvicko-map/src/components/App.tsx

```diff
@@ -61,6 +61,17 @@ export const App = () => {
   const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null);
   const [isMobile, setMobile] = useState<boolean | null>(null);
 
+  // set event listeners
+  useEffect(() => {
```

Comment:
Whole `useEffect` should be moved after the `stopAnimation` definition if it is using it.